execute/plugin.go

```diff
@@ -444,5 +479,19 @@ func (p *Plugin) supportsDestChain() (bool, error) {
 	return chains.Contains(p.cfg.DestChain), nil
 }
 
+func syncFrequency(configuredValue time.Duration) time.Duration {
+	if configuredValue.Milliseconds() == 0 {
+		return 10 * time.Second
+	}
+	return configuredValue
+}
+
+func syncTimeout(configuredValue time.Duration) time.Duration {
+	if configuredValue.Milliseconds() == 0 {
+		return 3 * time.Second
+	}
+	return configuredValue
+}
```

Comment:
Worth making these common or would the defaults be different?

Reply:
could be different, but atm they should be the same, since they serve the same purpose